Src/HALALMock/Services/Communication/SPI/SPI.cpp

```diff
@@ -1,16 +1,16 @@
-/*
- * SPI.hpp
- *
- *  Created on: 5 nov. 2022
- *      Author: Pablo
- */
-
 #include "HALALMock/Services/Communication/SPI/SPI.hpp"
 
+#include <netinet/in.h>
+#include <sys/socket.h>
+#include <unistd.h>
+
 #include "HALALMock/Models/MPUManager/MPUManager.hpp"
+#include "HALALMock/Services/SharedMemory/SharedMemory.hpp"
 
 #ifdef HAL_SPI_MODULE_ENABLED
 
+#define SPI_PORT_BASE 2000
```

Comment:
Now that all the services are going to start using sockets for everything i see just right to have something that unifies all the ports somewhere, something like a piece of code that has all the hardcoded ports for each of the services and checks for possible collisions or problems with the port already being used by the OS

Reply:
At least having a sys_config file with all of this parameters somewhere, there are going to be lots of parameters that it would be helpful to have a single place to change everything

Reply:
This is not strictly related to this PR, is more a structure matter. Please, discuss with @g0nz4I0 and @jmaralo

Reply:
Ideally we could have a small GUI to configure all the settings, and then generate a header file for C++ and another generated one for the python tests, something like the runes but more aimed to the simulator.